static/app/views/settings/organizationDeveloperSettings/index.tsx

```diff
@@ -59,20 +60,31 @@ class OrganizationDeveloperSettings extends AsyncView<Props, State> {
 
   renderInternalIntegrations() {
     const {orgId} = this.props.params;
+    const {organization} = this.props;
     const integrations = this.state.applications.filter(
       (app: SentryApp) => app.status === 'internal'
     );
     const isEmpty = integrations.length === 0;
 
+    const permissionTooltipText = t(
+      'Manager or Owner permissions required to add an internal integration.'
+    );
+
     const action = (
-      <Button
-        priority="primary"
-        size="small"
-        to={`/settings/${orgId}/developer-settings/new-internal/`}
-        icon={<IconAdd size="xs" isCircled />}
-      >
-        {t('New Internal Integration')}
-      </Button>
+      <Access organization={organization} access={['org:integrations']}>
```

Comment:
@MeredithAnya If we are excluding admins, I don't think this will work since admins have the `org:integrations` scope.

Reply:
oh yeah this needs to be `org:write`